src/Microsoft.Extensions.Configuration.UserSecrets/ConfigurationExtensions.cs

```diff
@@ -40,7 +39,7 @@ public static IConfigurationBuilder AddUserSecrets<T>(this IConfigurationBuilder
         /// </summary>
         /// <param name="configuration"></param>
         /// <returns></returns>
-        [Obsolete("This method is obsolete and will be removed in a future version. The recommended alternative is Microsoft.Extensions.Configuration.ConfigurationExtensions.AddUserSecrets<T>() or Microsoft.Extensions.Configuration.ConfigurationExtensions.AddUserSecrets(string userSecretsId).")]
+        [Obsolete("This method is obsolete and will be removed in a future version. The recommended alternative is Microsoft.Extensions.Configuration.ConfigurationExtensions.AddUserSecrets(this IConfigurationBuilder configuration, string userSecretsId).")]
```

Comment:
Did @eilon agree we could use the calling syntax instead of the full signature of the recommending method?